### PR TITLE
Build XC x86_64 module with CHPL_LIB_PIC=pic support.

### PR DESCRIPTION
**Pull request by jhoole**
Adding pic support into the module build in order to support python interoperability
on Cray XCs with x86 64 architecture. We skip builds for pic when communication
and launcher are not none since currently this only runs on the login node.

Tested manual build on Cicero and completed successfully.

> 2019-02-12 15:48:07,237 [INFO] All 864 configurations finished in 9511.975 seconds.

### CODE REVIEW

File: util/build_configs/cray-internal/setenv-xc-x86_64.bash [hunk 1]
```diff
@@ -266,7 +268,19 @@ else
     log_debug "with config=$BUILD_CONFIGS_CALLBACK"
 
     # Exit immediately to skip (avoid building) unwanted Chapel configs
-
+
```
Comment (ronawho) [inline util/build_configs/cray-internal/setenv-xc-x86_64.bash]: Could use a comment describing why we're skipping pic support for other configs.

**Comment by ronawho** [comment]
Title should probably say "Build the XC x86 module with ..."

It would also be helpful to give some context in the PR description for those not familiar with the motivation for this.

**Comment by ronawho** [comment]
> **continuous-integration/travis-ci/push ** — The Travis CI build failed

I'm like 8% sure (that's pretty high for me, I only go up to 10%) that this is because there was a branch named `RAD-828` at some point on chapel-lang, but it's been deleted now. I think you can ignore this failure.

**Comment by jhoole** [comment]
> This looks good to me. Do we have a sense for how much time the module build took/would take with this change versus before?

It took slightly under three hours when I ran it via command line, I don't know how long it took before.

**Comment by jhoole** [comment]
> Title should probably say "Build the XC x86 module with ..."
> 
> It would also be helpful to give some context in the PR description for those not familiar with the motivation for this.

Do you guys have a protocol for what should go in PR descriptions?

**Comment by ronawho** [comment]
> Do you guys have a protocol for what should go in PR descriptions?

https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/ContributorInfo.rst#final-merge-message

**Pull request merged**